sitemedia/scss/components/_pagination.scss

```diff
@@ -110,4 +110,11 @@ nav.pagination {
         font-family: "Phosphor";
         content: "\f044"; // phosphor arrow-right icon
     }
+    // Hebrew and Arabic reversed arrows
+    &[dir="rtl"] .prev::before {
+        content: "\f044"; // phosphor arrow-right icon
+    }
+    &[dir="rtl"] .next::after {
+        content: "\f03b"; // phosphor arrow-left icon
+    }
```

Comment:
will this be a problem if we're able to move the text-direction up to a higher level in the page?

Reply:
Yes, the `[dir="rtl"]` selector will need to match whatever level that attribute is set. Good catch!